src/Opdex.Platform.WebApi/Controllers/WalletsController.cs

```diff
@@ -93,6 +97,7 @@ public WalletsController(IApplicationContext context, IMapper mapper, IMediator
     /// <returns>Address balance summary.</returns>
     [HttpPost("{address}/balance/{token}")]
     [Authorize]
+    [CacheUntilNextBlock(CacheType.Public)]
```

Comment:
Should this be private since there's an `Authorize` attribute here?

Reply:
Private indicates the response might change per user. Any authorized user that calls this endpoint will get the same result. Potentially we could even remove the authorize attribute as it's not using any data from the token.

Reply:
Makes sense, I kept it authorized only so users can't spam this endpoint with addresses from others, mainly so we're not reaching out to Cirrus if someone wants to hit this any # of times for other addresses within rate limits.

Reply:
fwiw this should no longer be much of a problem as the API server _should_ itself cache the response for 16 seconds